packages/braid-design-system/src/lib/components/Inline/Inline.docs.tsx

```diff
@@ -93,17 +93,19 @@ const docs: ComponentDocs = {
       ),
       Example: () =>
         source(
-          <Stack space="medium" dividers align="center">
+          <Stack space="medium" align="center">
```

Comment:
The `align="center"` here makes the dividers disappear. Wrapping in Box width full fixes, or there might be a cleaner way. Maybe worth calling out in the migration guide too?

Reply:
I think this will be working when we get gap Stack though?

Reply:
As discussed we have improved `Divider` to prevent it collapsing when its a flex item and added it to this PR too.